airflow/operators/s3_to_mysql.py

```diff
@@ -0,0 +1,65 @@
+from airflow.hooks.mysql_hook import MySqlHook
+from airflow.models import BaseOperator
+from airflow.providers.aws.hooks.s3 import S3Hook
+from airflow.utils.decorators import apply_defaults
+
+
+class S3ToMySqlTransfer(BaseOperator):
+    """
+    Loads a file from S3 into a MySQL table.
+
+    :param s3_source_key: The path to the file (S3 key) that will be loaded into MySQL.
+    :type s3_source_key: str
+    :param mysql_table: The MySQL table into where the data will be sent.
+    :type mysql_table: str
+    :param delimiter: The delimiter for the file.
+    :type delimiter: str
+    :param header_rows: This parameter defines the number of header rows in the input file.
+    :type header_rows: int
+    :param aws_conn_id: The S3 connection that contains the credentials to the S3 Bucket.
+    :type aws_conn_id: str
+    :param mysql_conn_id: The MySQL connection that contains the credentials to the MySQL data base.
+    :type mysql_conn_id: str
+    """
+
+    template_fields = ('s3_source_key',)
+    template_ext = ()
+    ui_color = '#f4a460'
+
+    @apply_defaults
+    def __init__(self,
+                 s3_source_key,
+                 mysql_table,
+                 *args,
+                 delimiter=',',
+                 header_rows=1,
+                 aws_conn_id='s3_default',
+                 mysql_conn_id='mysql_default',
+                 **kwargs):
+        super(S3ToMySqlTransfer, self).__init__(*args, **kwargs)
+        self.s3_source_key = s3_source_key
+        self.mysql_table = mysql_table
```

Comment:
are you assuming the table is in the same schema as set in the connection?
what if it's not?

Reply:
Yes, I only tried it with a table in the schema which is defined in the connection.
Otherwise I think it will raise an error. But I can check that. Thanks :)